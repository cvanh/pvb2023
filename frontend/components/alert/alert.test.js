import renderer from "react-test-renderer";
import Alert from "./alert";
// import Link from '../Link';

// it('changes the class when hovered', () => {
//   const component = renderer.create(
//     <Link page="http://www.facebook.com">Facebook</Link>,
//   );
//   let tree = component.toJSON();
//   expect(tree).toMatchSnapshot();

//   // manually trigger the callback
//   renderer.act(() => {
//     tree.props.onMouseEnter();
//   });
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();

//   // manually trigger the callback
//   renderer.act(() => {
//     tree.props.onMouseLeave();
//   });
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();
// });

it("alert should show", () => {});

describe("alert should work", () => {
  it("should match", () => {
    const component = renderer.create(<Alert preview={true} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
