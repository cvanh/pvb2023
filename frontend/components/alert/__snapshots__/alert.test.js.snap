// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alert should work should match 1`] = `
<div
  className="sc-beqWaB jAXxAP"
>
  This is a page preview.
  <a
    href="/api/exit-preview"
  >
    Click here
  </a>
  to exit preview mode.
</div>
`;
